lib/index.js

```diff
@@ -1,16 +1,89 @@
-'use strict'
-
-var integration = require('@segment/analytics.js-integration')
+'use strict';
 
+var integration = require('@segment/analytics.js-integration');
+var useHttps = require('use-https');
+var reject = require('reject');
 /**
  * Expose `Wigzo` integration
  */
 
 var Wigzo = module.exports = integration('Wigzo')
+  .global('wigzo')
+  .option('orgToken', '')
+  .tag('tracker', '<script src="https://tracker.wigzopush.com/wigzo.js">')
+  .tag('https_manifest', '<link rel="manifest" href="/gcm_manifest.json">')
```

Comment:
i just double checked and it looks like our `this.load()` function will not be able to inject `<link>` tags. What is this for? If this is critical to the integration's set of features, I'd need to make a new release to the `analytics.js-integration` repo